v3/lints/rfc/lint_dnsname_label_too_long.go

```diff
@@ -29,8 +29,8 @@ func init() {
 		Name:          "e_dnsname_label_too_long",
 		Description:   "DNSName labels MUST be less than or equal to 63 characters",
 		Citation:      "RFC 1035",
-		Source:        lint.CABFBaselineRequirements,
-		EffectiveDate: util.CABEffectiveDate,
+		Source:        lint.RFC1035,
```

Comment:
So as you pointed out this might be a bit awkward, and I actually ran into this exact same issue myself in #586 .

My take on this is that we are citing 5280 which, in turns, "imports" and makes reference to 1035.

I suppose another way to say it is that the reason why this lint was written to begin with was not because of RFC1035, per se, but because RFC5280 invoked RFC1035. Only then did this character length become of interest to us.

@cpu and @zakird of course you folks have a stronger historical context than I do on the philosophy/semantics of `Source` so ya'lls two cents would carry significant weight on this.

Reply:
IMO, I prefer your interpretation. Perhaps additional context for why 5280 included it could be added to the description (such as `ctx. RFC1035`?